### PR TITLE
Buffing Bounty Rewards, Rebalancing Bounties for Security Roles

### PR DESCRIPTION
**Pull request by SirSwagmeyer**
## About The Pull Request

Increases the space credit reward for a good variety of departments that had woefully underfunded or underwhelming bounties and rewards. Gave Detectives security bounties. Secmeds get medical bounties too now.

## Why It's Good For The Game

Providing actionable rewards for bounties will incentivize players in the community other than medical players to do their bounties and bring more funding to cargo. 

Bounties are heavily underutilized in a majority of departments and heavily utilized in one or two departments. 
Security bounties net you around 120 credits for a handful of handcuffs, while medical gets thousands in the wallet for a bodybag full of synth organs. They both cost next to nothing to produce at roundstart. 

There's a clear imbalance to the bounty system, and I believe bringing the currently underwhelming bounty rewards for the other departments up to their level will make everyone happy. 

## Proof Of Testing

[Example of the improved bounty rewards. Ran it through locally, and it worked fine. 
](https://cdn.discordapp.com/attachments/1061369280572166164/1261562589587177512/image.png?ex=66936938&is=669217b8&hm=0ab84c6bffc5352e8acc20e4744d7b7c47ea8cdacfb60b9e4e7eb0d03e690c39&)

## Changelog

:cl:
add: Increased bounty multipliers for the following:
-  Engineers - emitters, borg chargers, SMES units, field generators, tesla coils.
- Robotics - All mechs. 
- Chemistry - Simple and rare chems.
- Security - Rechargers, Earmuffs, Handcuffs, N-Spect Reports
- Science - Modular Machines, Genetics Disability Mutators, all Refined Anomaly Cores

add: Added Security bounties to the Detective role. 
add: Added Medical bounties to the Security Medic role.
balance: Increased the quantity requirements for Security bounties and science's Modular laptops. 

/:cl:

### CODE REVIEW

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
modularize the bounty changes

File: modular_zubbers/code/modules/security/secmed/security_medic.dm [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
 	display_order = JOB_DISPLAY_ORDER_SECURITY_MEDIC
 	bounty_types = CIV_JOB_SEC
+	bounty_types = CIV_JOB_MED
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/security/secmed/security_medic.dm:27]: You can either have one or the other

File: modular_zubbers/code/modules/cargo/bounties/engineering.dm [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/datum/bounty/item/engineering/emitter
+	name = "Emitter"
+	description = "We think there may be a defect in your station's emitter designs, based on the sheer number of delaminations your sector seems to see. Ship us one of yours."
+	reward = CARGO_CRATE_VALUE * 8
+	wanted_types = list(/obj/machinery/power/emitter = TRUE)
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/cargo/bounties/engineering.dm:5]: When making overrides, you only need the variable you're changing
```suggestion
/datum/bounty/item/engineering/emitter
	reward = CARGO_CRATE_VALUE * 8
```

File: code/modules/jobs/job_types/detective.dm
Comment (Majkl-J) [inline code/modules/jobs/job_types/detective.dm:1]: Needs bubber edit comment or an override

File: code/modules/cargo/bounties/security.dm
Comment (Majkl-J) [inline code/modules/cargo/bounties/security.dm:1]: Needs to be made into overrides

File: modular_zubbers/code/modules/cargo/bounties/engineering.dm
Comment (Majkl-J) [inline modular_zubbers/code/modules/cargo/bounties/engineering.dm:1]: File is not included in the dme -> has no effect

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Making most of these into modular overrides and adding the files to the tgstation.dme reccomended

**Comment by Majkl-J** [comment]
staled out

**Pull request closed without merging**